### PR TITLE
update README that includes the details how to setup the project

### PR DESCRIPTION
**Pull request by marcinsawicki**
Resolves: #352 

## Description
Update of the README file, which includes the details related to instructions on how to setup the project.

## Storybook
null

## Checklist

**Obligatory:**

- [x] Self-review
- [ ] Unit & integration tests
- [ ] Storybook cases
- [ ] Design review
- [ ] Functional (QA) review

**Optional:**

- [ ] Accessibility cases (keyboard control, correct HTML markup, etc.)

### CODE REVIEW

**Review by TomaszRatajczyk — APPROVED** [review]

**Comment by MichalPaszowski** [comment]
I think I would add mention about how to work with the monorepo, why we have divided packages in that way and not the other,  maybe drop a word or two about lerna and how we use it. How to add a new package, how to bootstrap it etc.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 [See the Storybook](https://v1--613a8e945a5665003a05113b.chromatic.com/)
 
+# How to install?
+
+To run the project, first You need to use a the tool for managing the node version, which is convenient for you.
+We suggest using [Volta](https://volta.sh/).
+
+Required version of node is `16.13.2` and npm `8.1.0`.
```
Comment (mreszke) [inline README.md:10]: how I can easily switch to this version? Is there some file like `.nvmrc` with the node version so I can switch using one command like `nvm use` when using nvm?

Comment (sgraczyk) [reply README.md:10]: We've configured the project to use volta, we could also introduce `.nvmrc` file if you think it's necessary.

Comment (mreszke) [reply README.md:10]: ok, I just read the documentation, and I discovered that node version is changed automatically, so there is no need to do something like `nvm use` every time you change the project 👍 
I think it's worth mentioning that

**Pull request merged**